kubernetes_yaml/model_dowloader/container_file/download_hf_model.py

```diff
@@ -86,6 +89,13 @@ def save_model_to_s3(local_model_path: Union[str, Path], s3_model_path: str):
 
     return s3_path
 
+# Example usage with verbosity
+save_model_to_s3(
```

Comment:
We should probably just comment this out, since it will be invoked automatically when the entire script is run as-is